odoo/cli/command.py

```diff
@@ -46,7 +50,7 @@ def main():
 
     # TODO: find a way to properly discover addons subcommands without importing the world
     # Subcommand discovery
-    if len(args) and not args[0].startswith("-"):
+    if len(args) > 0 and not args[0].startswith("-"):
```

Comment:
useless change

Reply:
All the changes in this PR are cosmetic (this one makes intent more explicit in my opinion). I didn't know yet if it was appreciated to do such changes as one goes through code or not. I now understand that it is not and will revert such changes.